pkg/sql/logictest/testdata/logic_test/drop_index

```diff
@@ -464,3 +464,40 @@ t1_96731    t1_96731_pkey    PRIMARY KEY      PRIMARY KEY (i ASC)  true
 
 statement ok
 DROP TABLE t1_96731, t2_96731;
+
```

Comment:
Can we use an easier-to-understand name for those tables/columns in the logic test? Like we discussed, the following will be minimal repro:
```
create table t1 (i int primary key, j int);
create unique index t1_j_key on t1(j);

create table t3 (j int, i int primary key);
create table t2 (i int primary key, j int, constraint "fk_j_t1_i" foreign key (j) references t1(i), constraint "fk_j_t3_i" foreign key (j) references t3(i);

drop index t1_j_key;
```

Reply:
Let me adjust the names a bit, but I'm avoiding t1, etc.. because we get name collisions in these tests